### ISSUE TITLE
`@stylistic/function-parentheses-newline-inside` reports an opening parenthesis an inline comment already follows correctly

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#141]
Found by the sweep #115 is blocked on, which looks for a rule reading the text of an inline comment as code before the values a syntax prints start being read.

The `always` options ask for a line break after the opening parenthesis, and a comment may stand between the two: the rule walks past it and counts the line break behind it, which is what `getCheckBefore` is for. It walks past a block comment and stops at an inline one, since `postcss-value-parser` knows nothing of a comment opened by a double slash and hands the rule two division signs and a word instead of a comment. The line break closing that comment is then never reached, and the rule reports a function that already reads the way the option asks.

### What minimal example or steps are needed to reproduce the bug?

```less
a { t: translate( // c
  1px, 2px); }
```

### What minimal configuration is needed to reproduce the bug?

```json
{
  "customSyntax": "postcss-less",
  "plugins": "@stylistic/stylelint-plugin",
  "rules": {
    "@stylistic/function-parentheses-newline-inside": "always"
  }
}
```

### What did you expect to happen?

Nothing about the opening parenthesis. The same file with a block comment in place of the inline one draws no warning there, and the two say the same thing about where the line break stands.

### What actually happened?

```
1:19  Expected newline after "(" (@stylistic/function-parentheses-newline-inside)
```

`always-multi-line` says the same in its own words. The closing parenthesis is not affected: the whitespace behind the last argument is reached whether the comment in front of it is walked past or stopped at.

The fix that follows the warning writes a line break of its own in front of the comment, so a function that already read the way the option asks comes out reformatted:

```less
a { t: translate(
 // c
  1px, 2px
); }
```

### Do you have a proposal to fix the bug?

`getCheckBefore` walks the arguments and adds up the whitespace it finds until something that is neither whitespace nor a comment stops it. The nodes standing inside an inline comment have to be walked past there as well, and `findInlineCommentSpans` says where those are.

This blocks #115. Under `postcss-scss` the rule reads the copy of the value whose comments the syntax has already rewritten into block comments, so only `postcss-less` reports it today; handing the rule the copy that syntax prints makes the same file report under both.
